src/main/java/org/folio/search/converter/LanguageConfigConverter.java

```diff
@@ -0,0 +1,27 @@
+package org.folio.search.converter;
+
+import java.util.UUID;
+import lombok.experimental.UtilityClass;
+import org.folio.search.domain.dto.LanguageConfig;
+import org.folio.search.model.config.LanguageConfigEntity;
+
+@UtilityClass
```

Comment:
it is experimental feature, I guess in that case it can be replaced with just noArgConstructor

Reply:
Replaced with noArgsConstructor.